dart/lib/src/sentry_options.dart

```diff
@@ -261,6 +261,17 @@ class SentryOptions {
   /// Send statistics to sentry when the client drops events.
   bool sendClientReports = true;
 
+  /// If enabled, scope observers will be called when mutating scope.
+  bool enableScopeSync = true;
```

Comment:
We could link the docs to the `scopeObservers` property as well.

Search for `enableScopeSync` in the code, there are some TODOs, we can remove such comments since its added now.